### PR TITLE
Ownership change for SIG-VMware subprojects

### PR DESCRIPTION
**Pull request by frapposelli**
In preparation for the upcoming SIG folding (July 12th), SIG VMware subproject are being distributed across SIG Cluster Lifecycle and SIG Cloud Provider.

`cluster-api-provider-vsphere` is transferred over to sig-cluster-lifecycle ([ml message](https://groups.google.com/forum/#!topic/kubernetes-sig-cluster-lifecycle/CxyeaqI-dU0))
`vsphere-csi-driver` is transferred over to sig-cloud-provider (as determined during the sig folding plan).

Meetings are moved as well.

/cc @justinsb @luxas @timothysc @andrewsykim @hogepodge @jagosan

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -2240,26 +2266,10 @@ sigs:
...
-    tz: PT (Pacific Time)
-    frequency: bi-weekly
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
-    archive_url: https://docs.google.com/document/d/1jQrQiOW75uWraPk4b_LWtCTHwT7EZwrWWwMdxeWOEvk/edit?usp=sharing
-    recordings_url: https://www.youtube.com/playlist?list=PLutJyDdkKQIovV-AONxMa2cyv-_5LAYiu
   - description: Regular SIG Meeting
     day: Thursday
     time: "11:00"
-    tz: PT (Pacific Time)
+    tz: PT (Pacific Time)x
```
Comment (andrewsykim) [inline sigs.yaml]: typo?

Comment (luxas) [inline sigs.yaml]: unexpected?

Comment (frapposelli) [reply sigs.yaml]: good catch 👍

Comment (frapposelli) [reply sigs.yaml]: good catch 👍

**Comment by andrewsykim** [comment]
/approve
/lgtm

Thanks for helping make the transition smoother!

**Comment by andrewsykim** [comment]
xref: https://github.com/kubernetes/steering/issues/65

**Pull request merged**